pipelines/build/common/config_regeneration.groovy

```diff
@@ -301,55 +301,50 @@ class Regeneration implements Serializable {
             /*
             * Stage: Check that the pipeline isn't in inprogress or queued up. Once clear, run the regeneration job
             */
-            if (jobRootDir.contains("pr-tester")) {
-                // No need to check if we're going to overwrite anything for the PR tester
-                context.println "[SUCCESS] Don't need to check if the pr-tester is running. Running regeneration job..."
-            } else {
-                context.stage("Check $javaVersion pipeline status") {
-
-                    // Get all pipelines
-                    def getPipelines = queryAPI("${jenkinsBuildRoot}/api/json?tree=jobs[name]&pretty=true&depth1")
+            context.stage("Check $javaVersion pipeline status") {
 
-                    // Parse api response to only extract the relevant pipeline
-                    getPipelines.jobs.name.each{ pipeline ->
-                        if (pipeline.contains("pipeline") && pipeline.contains(versionNumbers[0])) {
-                            Integer sleepTime = 900
-                            Boolean inProgress = true
+                // Get all pipelines
+                def getPipelines = queryAPI("${jenkinsBuildRoot}/api/json?tree=jobs[name]&pretty=true&depth1")
 
-                            while (inProgress) {
-                                // Check if pipeline is in progress using api
-                                context.println "[INFO] Checking if ${pipeline} is running..." //i.e. openjdk8-pipeline
+                // Parse api response to only extract the relevant pipeline
+                getPipelines.jobs.name.each{ pipeline ->
+                    if (pipeline.contains("pipeline") && pipeline.contains(versionNumbers[0])) {
+                        Integer sleepTime = 900
+                        Boolean inProgress = true
 
-                                def pipelineInProgress = queryAPI("${jenkinsBuildRoot}/job/${pipeline}/lastBuild/api/json?pretty=true&depth1")
+                        while (inProgress) {
+                            // Check if pipeline is in progress using api
+                            context.println "[INFO] Checking if ${pipeline} is running..." //i.e. openjdk8-pipeline
 
-                                // If query fails, check to see if the pipeline been run before
-                                if (pipelineInProgress == null) {
-                                    def getPipelineBuilds = queryAPI("${jenkinsBuildRoot}/job/${pipeline}/api/json?pretty=true&depth1")
+                            def pipelineInProgress = queryAPI("${jenkinsBuildRoot}/job/${pipeline}/lastBuild/api/json?pretty=true&depth1")
```

Comment:
I was expecting the query to return true or false here, but I see that it's either a result (true) or null (false).  This doesn't need to change, but I'm making the observation for future coding style (converting result to a true/false status helps readers later on).

Reply:
Again, I haven't changed that line but I can see how that would be better on paper. A future change could be implemented to introduce that